lib/eslint-patch.js

```diff
@@ -1,13 +1,22 @@
 "use strict";
 
 const Plugins = require("eslint/lib/config/plugins");
+const ModuleResolver = require("eslint/lib/util/module-resolver");
+
+const ConfigFile = require("eslint/lib/config/config-file");
 
 const Config = require("eslint/lib/config");
 const ConfigUpgrader = require("./config_upgrader");
 
-const docs = require("./docs");
-
-module.exports = function patch(eslint) {
+module.exports = function patch() {
+  const resolve = ModuleResolver.prototype.resolve;
+  ModuleResolver.prototype.resolve = function(name, path) {
+    try {
+      return resolve.apply(this, [name, path]);
+    } catch(e) {
```

Comment:
@maxjacobson Any suggestions about the message? I am feeling it would just duplicate what we do at [line 34](https://github.com/codeclimate/codeclimate-eslint/pull/204/files#diff-2daeb06199eb4f1fdf8cd0fb63c70b21R34).